archivy/cli.py

```diff
@@ -28,11 +28,22 @@
 from archivy.check_changes import Watcher
 
 
+def show_license(ctx, param, value):  # noqa:
+    """Prints the license of the utility"""
+    if not value or ctx.resilient_parsing:
+        return
+    click.echo(__doc__)
+    ctx.exit()
+
+
 def create_app():
     return app
 
 
 @click.group(cls=FlaskGroup, create_app=create_app)
+@click.option('--license', '--lic', is_flag=True,
```

Comment:
I'm not sure we should add this. I don't see the point of adding a `license` option for each command.

Reply:
Hmm, I tried running this, I did not have any problem. Did you actually try executing it? 
If it gets applied to the subcommands too, then I would have to apply the same fix to `guiscrcpy` too.

Reply:
You are right I mixed things up.

This is a good idea for a new option :+1: